### PR TITLE
React homework week 3

### PR DESCRIPTION
**Pull request by myshuker**

### CODE REVIEW

File: react/week3/homework/src/App.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    fetchTodo().then(body=>{
+      this.setState({ 
+        todos: body
+      })
+    })
+  }
+
+  deleteItem =(id) =>{
+    console.log(id);
+    let items = this.state.todos.filter((item) => {
```
Comment (braedongough) [inline react/week3/homework/src/App.js:26]: Always use const unless you expect to reassign the variable.

File: react/week3/homework/src/App.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    item.id = Math.random();
+    let items = this.state.todos;
+    items.push(item);
+    this.setState({ items });
+  };
+
+  
+  
+/* 
+  editItem = (id) => {
```
Comment (braedongough) [inline react/week3/homework/src/App.js:61]: It's good practise to delete unused comments and clean up your code before submission!

File: react/week3/homework/src/component/TodoItems.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const TodoItems = (props) => {
+  //console.log(props);
+  const { items, deleteItem, todoCompleted, editItem, updateItem } = props;
```
Comment (braedongough) [inline react/week3/homework/src/component/TodoItems.js:5]: Nice use of destructring!

File: react/week3/homework/src/App.js (hunk 1 above)
Comment (myshuker) [reply react/week3/homework/src/App.js:26]: Done !

**Pull request merged**